src/uwtools/drivers/chgres_cube.py

```diff
@@ -35,18 +35,26 @@ def namelist_file(self):
             input_files.append(base_file)
         if update_values := namelist.get(STR.updatevalues):
```

Comment:
Taking a peek in chgres_cube src code, it looks like these are the entries that should be combined with directories and files:

data_dir_input_grid + atm_files_input_grid (list or string)
data_dir_input_grid + grib2_file_input_grid
data_dir_input_grid + nst_files_input_grid
data_dir_input_grid + sfc_files_input_grid (list or string)
orog_dir_input_grid + orog_files_input_grid (list or string)
orog_dir_target_grid + orog_files_target_grid (list or string)

Reply:
Hm, it looks like its surprisingly honest about 'files' = list. Presumably `nst_files_input_grid` also takes a list, while `varmap_file` and `vcoord_file_target_grid` should be removed from the list check? I'll have to recheck the source too...

Reply:
Ah, I'm going to need to make these mutually required in the schema too